src/epm.rs

```diff
@@ -67,7 +67,7 @@ pub(crate) async fn update_metadata_constants(api: &SubxtClient) -> Result<(), E
 	const SIGNED_MAX_WEIGHT: EpmConstant = EpmConstant::new("SignedMaxWeight");
 	const MAX_LENGTH: EpmConstant = EpmConstant::new("MinerMaxLength");
 	const MAX_VOTES_PER_VOTER: EpmConstant = EpmConstant::new("MinerMaxVotesPerVoter");
-	const MAX_WINNERS: EpmConstant = EpmConstant::new("MinerMaxWinners");
+	const MAX_WINNERS: EpmConstant = EpmConstant::new("MaxWinners");
```

Comment:
Ok, this was set to work with older metadata right?

Smart move as this is just duplicated in the `MinerConfig` :P

Reply:
Maybe add a comment for that :)